src/System.Runtime/tests/System/Char.cs

```diff
@@ -641,6 +645,14 @@ public static void TestIsSurrogatePairStrInt()
         foreach (char hs in s_highSurrogates)
             foreach (char ls in s_nonSurrogates)
                 Assert.False(Char.IsSurrogatePair(hs.ToString() + ls, 0));
+
+        Assert.Throws<ArgumentNullException>(() => char.IsSurrogatePair(null, 0));
+
+        Assert.Throws<ArgumentOutOfRangeException>(() => char.IsSurrogatePair("abc", -1));
+
+        Assert.Throws<ArgumentOutOfRangeException>(() => char.IsSurrogatePair("abc", 4));
+
+        Assert.False(char.IsSurrogatePair("abc", 2)); // index + 1 < s.length always returns false.
```

Comment:
Could you clarify the comment? In this case, `index + 1 == s.Length`, right? But "bc" isn't a surrogate pair, so it returns false. Doesn't `index + 1 < Length` mean we will check the char pair since there's at least one more char after `index`?